ts/store/reducers/entities/services/index.ts

```diff
@@ -138,6 +140,10 @@ const getServices = (
         .filter(isDefined)
         .filter(service =>
           hasLocalization(service, servicesMetadata.byId, localization)
+        )
```

Comment:
Could we collapse all these filter predicates?

```typescript
const data = serviceIdsForOrg
        .map(id => services.byId[id])
        .filter(
          service =>
            isDefined(service) &&
            hasLocalization(service, servicesMetadata.byId, localization) &&
            isVisibleService(visibleServices, service)
        );
```
visibleServices can't be undefined, shouldn't it?

Reply:
yes, updated within https://github.com/teamdigitale/io-app/pull/1327/commits/e0c964cb9b83fe9048f1d5155a4e99a88c46e332